iron-menu-behavior.html

```diff
@@ -144,11 +144,18 @@
     _focusPrevious: function() {
       var length = this.items.length;
       var curFocusIndex = Number(this.indexOf(this.focusedItem));
+
       for (var i = 1; i < length + 1; i++) {
         var item = this.items[(curFocusIndex - i + length) % length];
+        var owner = Polymer.dom(item).getOwnerRoot() || document;
         if (!item.hasAttribute('disabled')) {
```

Comment:
is this detecting the cases where `[disabled]` _actually_ isn't focusable, or is _logically_ unfocusable?

because if it's _actually_, the `activeElement` check should handle this, right?